content/en/blog/_posts/2020-08-00-Moving-Forward-From-Beta/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: blog
+title: "Moving Forward From Beta"
+date: 2020-08-26
+slug: moving-forward-from-beta
+---
+
+**Author**: Tim Bannister, The Scale Factory
+
+In Kubernetes, features follow a defined
+[lifecycle](/docs/reference/command-line-tools-reference/feature-gates/#feature-stages).
+First, as the twinkle of an eye in an interested developer. Maybe, then,
+sketched in online discussions, drawn on the online equivalent of a cafe
+napkin. This rough work typically becomes a
+[Kubernetes Enhancement Proposal](https://github.com/kubernetes/enhancements/blob/master/keps/0001-kubernetes-enhancement-proposal-process.md#kubernetes-enhancement-proposal-process) (KEP), and
+from there it usually turns into code.
+
+For Kubernetes v1.20 and onwards, we're focusing on helping that code
+graduate into stable features.
+
+That lifecycle I mentioned runs as follows:
+
+![Alpha → Beta → General Availability](feature_stages.svg)
+
+Usually, alpha features aren't enabled by default. You turn them on by setting a feature
+gate; usually, by setting a command line flag on each of the components that use the
+feature.
+
+(If you use Kubernetes through a managed service offering such as AKS, EKS, GKE, etc then
+the vendor who runs that service may have decided what feature gates are enabled for you).
+
+There's a defined process for graduating an existing, alpha feature into the beta phase.
+This is important because beta features are enabled by default, with the feature flag still
```

Comment:
```suggestion
This is important because **beta features are enabled by default**, with the feature flag still
```
Not a requirement or anything, but it might be worthwhile to specifically highlight this. I've run into folk who didn't know this was the current policy and it also sorta begins the focus of the rest of the doc.